charts/node/values.yaml

```diff
@@ -54,58 +89,102 @@ ingress:
   #    hosts:
   #      - chart-example.local
 
-# Add extra labels on pods and services
-#extraLabels:
-#  type: rpc
 
+## Deploy a substrate node
+## ref: https://docs.substrate.io/v3/getting-started/overview/
```

Comment:
It's strange because this link doesn't provide any help to setup your node. Maybe we can remove it.